SofaKernel/modules/SofaHelper/src/sofa/helper/RandomGenerator.cpp

```diff
@@ -43,7 +43,7 @@ namespace sofa
 namespace helper
 {
 
-const unsigned long RandomGenerator::RANDOM_BASE_MAX = 4294967295U;
+constexpr unsigned long RandomGenerator::RANDOM_BASE_MAX = 4294967295U;
```

Comment:
RANDOM_BASE_MAX is declared static in the header.
you could move the initialization in the .h (inline static initialization) I suppose